manifests/d/dot3labs/Cube2/2020.11.29/dot3labs.Cube2.yaml

```diff
@@ -0,0 +1,30 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/microsoft/winget-cli/master/schemas/JSON/manifests/v1.0.0/manifest.singleton.1.0.0.json
+PackageIdentifier: dot3labs.Cube2
+PackageVersion: 2020.11.29
+PackageName: 'Cube 2: Sauerbraten'
+Publisher: dot 3 labs LLC
+Author: Aardappel
+Moniker: Sauerbraten
+PublisherUrl: https://strlen.com/dot3labs/
+CopyrightUrl: http://sauerbraten.org/README.html
+PublisherSupportUrl: http://cubeengine.com/
+License: zlib
+LicenseUrl: http://sauerbraten.org/README.html
+Tags:
+- sauerbraten
+- cube2
+- game
+- fps
+- shooter
+- first person
+ShortDescription: Cube2:Sauerbraten is the successor to the Cube FPS Engine
+Description: 'Cube 2: Sauerbraten is a free multiplayer & singleplayer first person shooter, the successor of the Cube FPS. Much like the original Cube, the aim of this game is fun, old school deathmatch gameplay and also to allow map/geometry editing to be done cooperatively in-game.'
+PackageUrl: http://sauerbraten.org/
+Installers:
+- Architecture: x86
```

Comment:
```suggestion
- Architecture: x64
```

Reply:
Its a 2 in 1 installer...Would this change reflect that? it has both the x64 and x32. I believe. I haven't deconstructed the exe myself recently.